deploy.go

```diff
@@ -1257,7 +1318,8 @@ func writeConfig(config *CookieConfig) error {
 		return err
 	}
 	// #nosec G306
-	err = ioutil.WriteFile(configFile, data, 0o644)
+	configFile, _ := getConfigPath()
```

Comment:
variable already mentioned above then no need to get again

Reply:
The first `configFile, _ := getConfigPath()` at line 1296 is in method `loadConfig`, while `configFile, _ := getConfigPath()` at line 1321 is in method `writeConfig`